specs/agents/mobile/README.md

```diff
@@ -79,3 +78,10 @@ The span name should have the format: `<method> <host>`.
 | [`net.peer.name`](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/http/)  |  ---                  | `localhost`    | :x: no| Fallback field to derive HTTP-related fields if `http.url` field is not provided.|
 | [`net.peer.port`](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/http/)  |  ---                  | `3000`          | :x: no| Fallback field to derive HTTP-related fields if `http.url` field is not provided. |
 
+### Other Attributes
```

Comment:
I like this new table, I think it makes it clearer until OTel provides a definition for these kinds of attrs.